packages/utils/ActivityManager/image.js

```diff
@@ -40,8 +40,22 @@ export function setupImgMapping() {
         return src;
     };
 
-    ["DrawImageEx", "GLDrawImage", "DrawGetImage"].forEach(
-        (/** @type {"DrawImageEx" |"GLDrawImage"| "DrawGetImage"}*/ fn) =>
-            ModManager.progressiveHook(fn, 9).inject((args, next) => (args[0] = mapImgSrc(args[0])))
-    );
+    if (GameVersion === "R110") {
+        ["DrawImageEx", "GLDrawImage", "DrawGetImage"].forEach(
+            (/** @type {"DrawImageEx" |"GLDrawImage"| "DrawGetImage"}*/ fn) =>
+                ModManager.progressiveHook(fn, 9).inject((args, next) => (args[0] = mapImgSrc(args[0])))
+        );
+    } else { // R111
+        ModManager.hookFunction("ElementButton.CreateForActivity", 0, (args, next) => {
+            const button = next(args);
+            const img = button.querySelector("img.button-image");
+            if (img?.src) {
+                const idx = img.src.indexOf("Assets/");
+                if (idx !== -1) {
+                    img.src = mapImgSrc(decodeURI(img.src.slice(idx)));
+                }
+            }
+            return button;
+        });
```

Comment:
Would generally recommend hooking into the function and passing the image src along _before_ the button is actually created (similar to https://github.com/littlesera/LSCG/blob/cd942dd60a46e843f215948bfe630ac26a0dc44b/src/Modules/activities.ts#L199-L205), but this would require the activity/asset names to have a common component used for marking it as part of the echo bc mod collection.

Reply:
Since all the mapping here are added by `addActivityImageMapping`, so only need to refactor those to create a mapping from some Activity identitiy to the actual resouce url.